lib/Doctrine/DBAL/Migrations/Configuration/Configuration.php

```diff
@@ -638,7 +638,9 @@ public function getNextVersion()
     /**
      * Returns the version with the specified offset to the specified version.
      *
-     * @return string|null A version string, or null if the specified version
+     * @param $version string
```

Comment:
The type precedes the variable name. This goes for all param documentation in this PR.

Reply:
Weird are you sure ? Because most of those were auto generated by PHPStorm.

Reply:
See other examples in the same file.